{"name": "2. Проверка отображения заказа в истории заказов", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:552:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16", "trace": "self = <tests.test_order_feed.TestOrderFeed object at 0x000001DE7E161350>\ndriver = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"be285cb4-391b-45d8-b5b0-66ab6486d9ef\")>, login = {'email': 'test8915@test.com', 'token': None}\nmain_page = <pages.main_page.MainPage object at 0x000001DE7E3A9310>, order_feed_page = <pages.order_feed_page.OrderFeedPage object at 0x000001DE000742B0>\nprofile_page = <pages.profile_page.ProfilePage object at 0x000001DE7E3C4F50>\n\n    @allure.title(\"2. Проверка отображения заказа в истории заказов\")\n    def test_order_in_history(self, driver, login, main_page, order_feed_page, profile_page):\n        with allure.step(\"1. Создать и оформить заказ\"):\n            main_page.open()\n            bun = WebDriverWait(driver, 15).until(\n                EC.presence_of_element_located(MainPageLocators.FLUORESCENT_BUN)\n            )\n            constructor = WebDriverWait(driver, 15).until(\n                EC.presence_of_element_located(MainPageLocators.CONSTRUCTOR_AREA)\n            )\n            driver.execute_script(\"\"\"\n                const dataTransfer = new DataTransfer();\n                arguments[0].dispatchEvent(new DragEvent('dragstart', {dataTransfer}));\n                arguments[1].dispatchEvent(new DragEvent('drop', {dataTransfer}));\n            \"\"\", bun, constructor)\n>           WebDriverWait(driver, 15).until(\n                lambda d: d.find_element(*MainPageLocators.MAKE_ORDER_BUTTON).is_enabled()\n            ).click()\n\ntests\\test_order_feed.py:50: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"be285cb4-391b-45d8-b5b0-66ab6486d9ef\")>\nmethod = <function TestOrderFeed.test_order_in_history.<locals>.<lambda> at 0x000001DE7E3A96C0>, message = ''\n\n    def until(self, method: Callable[[WebDriver], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nE       NoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:552:5\nE       dom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\nvenv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py:101: TimeoutException"}, "steps": [{"name": "1. Создать и оформить заказ", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:552:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n", "trace": "  File \"C:\\Users\\windw\\PycharmProjects\\Diplom\\Diplom_3\\tests\\test_order_feed.py\", line 50, in test_order_in_history\n    WebDriverWait(driver, 15).until(\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^\n        lambda d: d.find_element(*MainPageLocators.MAKE_ORDER_BUTTON).is_enabled()\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    ).click()\n    ^\n  File \"C:\\Users\\windw\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 101, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1748199534366, "stop": 1748199556032}], "parameters": [{"name": "driver", "value": "'firefox'"}], "start": 1748199534366, "stop": 1748199556034, "uuid": "34e39c88-4e49-4544-a4f7-f6bd89ae915f", "historyId": "81a06631253827fc583a853136cfa523", "testCaseId": "faa93f48a6011244bddb3f3af3273d26", "fullName": "tests.test_order_feed.TestOrderFeed#test_order_in_history", "labels": [{"name": "feature", "value": "Лента заказов"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order_feed"}, {"name": "subSuite", "value": "TestOrderFeed"}, {"name": "host", "value": "Gemini_78"}, {"name": "thread", "value": "19464-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order_feed"}]}